### PR TITLE
Candlestick wick and labels

### LINKED ISSUE #269: Label positioning for candlestick
Candlestick labels are currently poorly positioned
screen shot 2016-06-25 at 6 46 05 pm

### PR DESCRIPTION
**Pull request by ebrillhart**
fixes #269 and #270 - fixes the label positioning for data objects in victory-candlestick and makes wick match candle fill unless stroke color is specified separately.

@boygirl

### CODE REVIEW

File: src/components/victory-candlestick/helper-methods.js [hunk 1]
```diff
@@ -27,8 +27,8 @@ export default {
       const labelStyle = this.getLabelStyle(style.labels, dataProps);
       const labelProps = {
         style: labelStyle,
-        x,
-        y: y - labelStyle.padding,
+        x: x - labelStyle.padding,
+        y: y - (labelStyle.padding * 2),
```
Comment (boygirl) [inline src/components/victory-candlestick/helper-methods.js]: I think this should be something in relation to` y1` if you want to labels to always appear higher than the top of the wick. I think you're just getting lucky by doubling the padding.

**Comment by boygirl** [comment]
@ebrillhart stroke / fill stuff works perfectly. Label positioning needs a little more love.

**Comment by ebrillhart** [comment]
@boygirl went with just using the label padding to push the label out a bit from the wick and up a bit from the body so that it didn't look squished (since if it's at y it'll stay right at the top of the body of the candle, which doesn't look great).

**Comment by boygirl** [comment]
@ebrillhart I'm happy with this now that the `padding * 2` is removed. Merging.

**Pull request merged**